src/main/osd/osd_elements.c

```diff
@@ -1469,7 +1469,7 @@ static void osdElementWarnings(osdElementParms_t *element)
             }
             #endif // USE_RX_LINK_QUALITY_INFO
         }
-        strncpy(pilotConfigMutable()->name, element->buff, MAX_NAME_LENGTH);
+        strncpy(pilotConfigMutable()->name, element->buff, MAX_NAME_LENGTH - 1);
```

Comment:
```
typedef struct pilotConfig_s {
    char name[MAX_NAME_LENGTH + 1];
    char displayName[MAX_NAME_LENGTH + 1];
} pilotConfig_t;
```

It is declared this way. So don't see the reason to shrink it by another byte. Do we need to null terminate string instead?

```
pilotConfigMutable()->name[MAX_NAME_LENGTH] = 0;
```

Reply:
Tested that, adding null termination does not help. I get the same `CRASH F...` truncation as before.